### PR TITLE
Changed Vagrantfile to use docker-compose.yml

### PR DESCRIPTION
**Pull request by vadimsh**
Vagrantfile: 
- "dcompose" script added 
- docker section removed except docker install and devopsloft_network create
 (works on default network)
- added shell commands to remove Ubuntu locks, placed before "apt-get update"

Docker-compose.yml :
- devopsloft_network commented out but can be added back if needed

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,33 @@
+version: "3.7"
```
Comment (noamgu) [inline docker-compose.yml]: I think we should template it using .env file

https://docs.docker.com/compose/environment-variables/#the-env-file

One again, feel fre

**Comment by lioramilbaum** [comment]
Have you used the docker provisioner for docker-compose?
For more information - https://www.vagrantup.com/docs/docker/configuration.html

File: Vagrantfile [hunk 2]
```diff
@@ -127,25 +136,18 @@ Vagrant.configure("2") do |config|
   config.vm.provision 'shell',
     inline: $set_environment_variables, run: "always"
   config.vm.provision "shell",
-    inline: "apt-get update; apt-get install -y mysql-client"
+    inline: "rm /var/lib/dpkg/lock-frontend; rm /var/lib/dpkg/lock; rm /var/cache/apt/archives/lock; apt-get update; apt-get install -y mysql-client"
```
[the remark this answers is not in the document]
Comment (vadimsh) [reply Vagrantfile]: I found the process named unattended-upgrade which sometimes causes locks, replaced removal of locks with this command:  kill $(pgrep 'unattended').

File: docker-compose.yml (hunk 1 above)
Comment (vadimsh) [reply docker-compose.yml]: I have added all parameters from .env to docker-compose.yml and tested with "vagrant up/destroy stage"

**Comment by vadimsh** [comment]
I haven't used docker-compose provisioner from Vagrant, just from the shell:
$dcompose = <<-SCRIPT
curl -L https://github.com/docker/compose/releases/download/1.24.0/docker-compose-$(uname -s)-$(uname -m) -o /usr/local/bin/docker-compose > /dev/null 2>&1
chmod +x /usr/local/bin/docker-compose > /dev/null 2>&1
set -a
source /vagrant/.env
docker-compose -f /vagrant/docker-compose.yml up -d --build --force-recreate
SCRIPT

**Comment by lioramilbaum** [comment]
Would you like to try it out?

**Comment by lioramilbaum** [comment]
Prometheus deployment is not related to this PR.

**Pull request closed without merging**